scrapers/javlibrary.yml

```diff
@@ -122,42 +122,216 @@ xPathScrapers:
+      #      - regex: V\*{5}ed\b
+      #        with: "Violated"
+      #      - regex: StepB\*{16}r\b
+      #        with: "StepBrother And Sister" # Not Sure
+      #      - regex: StepK\*{1}ds \b
+      #        with: "StepKids"
+      #      - regex: C\*{3}dhood\b
+      #        with: "Childhood"
+      #      - regex: V\*{6}e\b
+      #        with: "Violence"
+      #      - regex: C\*{3}dcare\b
+      #        with: "Childcare"
+      #      - regex: H\*{7}m\b
+      #        with: "Hypnotism"
+      #      - regex: M\*{4}tation\b
+      #        with: "Molestation"
+      #      - regex: M\*{4}ting\b
+      #        with: "Molesting"
+      #      - regex: F\*{3}e\b
+      #        with: "Force"
+      #      - regex: A\*{3}es\b
+      #        with: "Abuses"
+      #      - regex: A\*{3}e\b
+      #        with: "Abuse"
+      #      - regex: S\*{6}g\b
+      #        with: "Sleeping"
+      #      - regex: A\*{3}ed\b
+      #        with: "Abused"
+      #      - regex: D\*{2}g\b
+      #        with: "Drug"
+      #      - regex: A\*{5}ted\b
+      #        with: "Assaulted"
+      #      - regex: A\*{5}t
+      #        with: "Assault"
+      #      - regex: S\*{8}n\b
+      #        with: "Submission"
+      #      - regex: K\*{1}d\b
+      #        with: "Kid"
+      #      - regex: D\*{6}e\b
+      #        with: "Disgrace"
+      #      - regex: Y\*{8}ls\b
+      #        with: "Young Girls" # Not Sure
+      #      - regex: Y\*{8}l\b
+      #        with: "Young Girl" # Not Sure
+      #      - regex: H\*{2}t\b
+      #        with: "Hurt"
+      #      - regex: H\*{2}ts\b
+      #        with: "Hurts"
+      #      - regex: C\*{3}dren\b
+      #        with: "Children"
+      #      - regex: F\*{3}es\b
+      #        with: "Forces" # Not Sure
+      #      - regex: Chai\*{1}saw\b
+      #        with: "Chainsaw"
+      #      - regex: Lol\*{1}pop\b
+      #        with: "Lolipop"
+      #      - regex: K\*{4}pped\b
+      #        with: "Kidnapped"
+      #      - regex: K\*{4}pping\b
+      #        with: "Kidnapping"
+      #      - regex: B\*{5}p\b
+      #        with: "Bang Up" # Not Sure
+      #      - regex: D\*{3}ken\b
+      #        with: "Drunken"
+      #      - regex: Lo\*{2}ta\b
+      #        with: "Lolita"
+      #      - regex: CrumB\*{2}d\b
+      #        with: "CrumBled"
+      #      - regex: K\*{1}dding\b
+      #        with: "Kidding"
+      #      - regex: C\*{3}d\b
+      #        with: "Child"
+      #      - regex: C\*{5}y\b
+      #        with: "Cruelty"
+      #      - regex: H\*{9}n\b
+      #        with: "Humiliation"
+      #      - regex: D\*{6}eful\b
+      #        with: "Disgraceful"
+      #      - regex: B\*{3}d\b
+      #        with: "Blood"
+      #      - regex: U\*{7}g\b
+      #        with: "Unwilling"
+      #      - regex: HumB\*{2}d\b
+      #        with: "Humbled"
+      #      - regex: K\*{2}ler\b
+      #        with: "Keeler" # Not Sure
```

Comment:
Does Killer make more sense?

Reply:
Tbh i have no idea, here is the source: 
`https://www.r18.com/videos/vod/movies/detail/-/id=nsps00929/`

As a non native english, if this make more sense for you i will change 😄

Reply:
With a google search all cases with the id presentm as comment should be ok except this one , in various results `killer` is mentioned so that is probably better.